backend/database/entities/Game.ts

```diff
@@ -4,7 +4,10 @@ import { BaseEntity } from './BaseEntity';
 @Entity('games')
 export class Game extends BaseEntity {
 	@Column({ name: 'series_id', type: 'int', unsigned: true })
-	public seriesId: string;
+	public seriesId: number;
+
+	@Column({ name: 'image_id', type: 'int', unsigned: true, nullable: true })
```

Comment:
In case of nullable `imageId`, are we dealing with in response, or we have to add a check in front-end to be sure that `fileId` isn't null to not have application crash?

Reply:
Yup. In case there is no image for something just use the default placeholder image that you have.